casper/src/main/scala/coop/rchain/casper/rholang/syntax/RuntimeSyntax.scala

```diff
@@ -457,12 +469,12 @@ final class RuntimeOps[F[_]](private val runtime: RhoRuntime[F]) extends AnyVal
     )
 
     // Create return channel as first private name created in deploy term
-    val rand = Tools.unforgeableNameRng(deploy.pk, deploy.data.timestamp)
+    val rand = Blake2b512Random.apply(10)
```

Comment:
Why length 10? We now have 128 defaults. And why to explicitly write `apply`?
https://github.com/rchain/rchain/blob/6dd06305020f02f9fed99c7eda6cbd980fdb0b7d/rholang/src/main/scala/coop/rchain/rholang/interpreter/RhoRuntimeSyntax.scala#L42